src/data_scripts/read_data.py

```diff
@@ -0,0 +1,33 @@
+
+from google.cloud import storage
+import requests
+import pandas as pd
+from io import StringIO
+
+def get_data(url = "https://storage.googleapis.com/cloud-samples-data/ai-platform-unified/datasets/tabular/petfinder-tabular-classification.csv"):
```

Comment:
Make the url a static variable, named in CAPS and pass it in.

So something like `DATA_SOURCE = "https://storage.googleapis.com/cloud-samples-data/ai-platform-unified/datasets/tabular/petfinder-tabular-classification.csv"`.

Reply:
Got it. I will make that change